jinahub/indexers/searcher/HnswlibSearcher/tests/test_hnswlibsearcher.py

```diff
@@ -29,6 +29,23 @@ def metas(tmpdir):
     del os.environ['TEST_WORKSPACE']
 
 
+@pytest.mark.parametrize(['metric', 'is_distance'],
+                         [('l2', True), ('ip', True), ('cosine', True),
+                          ('l2', False), ('ip', False), ('cosine', False)])
+def test_metric(tmpdir, metric, is_distance):
+    metas = {'workspace': str(tmpdir), 'name': 'searcher', 'pea_id': 0, 'replica_id': 0}
+
+    indexer = HnswlibSearcher(dump_path=DUMP_PATH, top_k=TOP_K, metas=metas, metric=metric, is_distance=is_distance)
+    docs = DocumentArray([Document(embedding=np.random.random(7))])
+    indexer.search(docs, {})
+
+    for i in range(len(docs[0].matches) - 1):
```

Comment:
assert length of matches before. To make sure the `for` actually happens

Reply:
ah yes, sure